src/api/cloud/project/io/stream/cloud-project-io-stream.v6.service.js

```diff
@@ -0,0 +1,16 @@
+angular
+  .module('ovh-api-services')
+  .service('OvhApiCloudProjectIoStreamV6', ($resource) => {
+    const resource = $resource('/cloud/project/:serviceName/io/stream/:streamId', {
+      serviceName: '@serviceName',
+      streamId: '@streamId',
+    }, {
+      getStats: {
+        url: '/cloud/project/:serviceName/io/stream/:streamId/stats',
```

Comment:
No cache ?

Reply:
There is no need to handle the cache at this level.